app/matplotlib.ipynb

```diff
@@ -194,21 +190,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
-    "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, startangle=90)\n",
+    "plt.title(\"Market Share\")\n",
+    "plt.show()"
```

Comment:
In Task 5, the pie chart should display the numbers in the format '10.0%'. You can achieve this by using the `autopct` parameter in `plt.pie()`, e.g., `autopct='%1.1f%%'`.